packages/opds-web-client/src/components/Book.tsx

```diff
@@ -28,38 +28,41 @@ export default class Book<P extends BookProps> extends React.Component<P, void>
   }
 
   render(): JSX.Element {
+    const book = this.props.book;
     // Remove HTML tags from the summary to fit more information into a truncated view.
     // The summary may still contain HTML character entities and needs to be rendered as HTML.
-    let summary = (this.props.book && this.props.book.summary && this.props.book.summary.replace(/<\/?[^>]+(>|$)/g, " ")) || "";
-    const bookMedium = this.getMedium(this.props.book);
+    let summary = (book && book.summary && book.summary.replace(/<\/?[^>]+(>|$)/g, " ")) || "";
```

Comment:
What's the purpose of this line?

Reply:
Looks like it cleans up the summary.

Reply:
What kind of thing would there be that would need cleaning up?

Reply:
@aslagle ?

Reply:
This is in the compact list view where we're trying to fit as much info as possible. Some of the distributors put lots of line/paragraph breaks in their summaries which means hardly any text ends up in the compact view- sometimes it's just one line and then empty space. This strips out all of that formatting and collapses the text into a single paragraph, but if you click through to the book details page you can see the nicely formatted summary.